src/bin/vip-import-media.js

```diff
@@ -102,8 +102,14 @@ Are you sure you want to import the contents of the url?
 	.option( 'importIntermediateImages', 'Import intermediate image files', false )
 	.examples( examples )
 	.argv( process.argv, async ( args, opts ) => {
-		const { app, env, exportFileErrorsToJson, overwriteExistingFiles, importIntermediateImages } =
-			opts;
+		const {
+			app,
+			env,
+			exportFileErrorsToJson,
+			overwriteExistingFiles,
+			importIntermediateImages,
+			force,
```

Comment:
Traditionally, `--force` is also used to bypass warnings in the CLI. I'm wondering if we can use a name that conveys exactly what we're trying to "force" or rather, "skip" . Perhaps `'--skip-download-failure-details', '-S'` ? It's a mouthful but I would rather be precise and we would have the shortcode option as well.

Reply:
I think the global `--force` flag was meant to skip the confirmation that some commands require _before_ the command is actually run.

https://github.com/Automattic/vip-cli/blob/trunk/src/lib/cli/command.js#L582

Reply:
What if the user who is running a script wants the error reports to be downloaded without manual intervention?

Reply:
> What if the user who is running a script wants the error reports to be downloaded without manual intervention?

Same idea could be applied to that. I've edited my original comment, please check.

Reply:
@saroshaga adding the more specific flag is what I was going for in #2007 although I personally don't mind if `--force` auto-saves the log as well.

Reply:
@saroshaga Thanks for your input. Let's go with #2007 then. I will close this PR.